### PR TITLE
refactor: GEOT-278: rename nav component and notification component

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/ui/NavigationMain/index.tsx [hunk 1]
```diff
@@ -3,74 +3,86 @@ import styles from './styles.scss'
...
               label: 'My groups',
               icon: <IoIosKeypad />,
               url: '/my-groups',
             },
           ].map((value, index) => {
             return (
               <li className={styles.link} key={index}>
-                <Link href={value.url}>
-                  <a>
+                {value?.fn && (
```
Comment (ra8ga) [inline packages/ui/NavigationMain/index.tsx:60]: how to write it properly?

File: packages/app-andrew/src/components/commons/NavigationGroup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState, useEffect } from 'react'
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/commons/NavigationGroup/index.tsx:1]: NavigationGroups - current name is wrong

Comment (ra8ga) [reply packages/app-andrew/src/components/commons/NavigationGroup/index.tsx:1]: you mean with 's' at the end? 
it's navigation for single group

File: packages/app-andrew/src/components/commons/NotificationsSidebar/index.tsx [hunk 3]
```diff
@@ -4,10 +4,13 @@ import cx from 'classnames'
 import useKey from 'use-key-hook'
 import { Scrollbars } from 'react-custom-scrollbars'
 import { IoMdCloseCircle } from 'react-icons/io'
-import { Notification } from '@gtms/ui/Notification'
+import { NotificationLabel } from '@gtms/ui/NotificationLabel'
 import { INotificationsSidebarProps, baseUIQuery } from 'queries'
 import { NotificationIcons } from 'enums'
-import { INotification, deleteNotification } from '@gtms/state-notification'
+import {
+  INotificationLabel,
```
Comment (ra8ga) [inline packages/app-andrew/src/components/commons/NotificationsSidebar/index.tsx:11]: error

**Pull request closed without merging**